Makefile

```diff
@@ -10,6 +10,9 @@ default:
 angel:
 	@./mvnw clean test -Dversion.org.springframework.boot=1.2.8.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Angel.SR6
 
+edgware:
+	@./mvnw clean test -Dversion.org.springframework.boot=1.5.8.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Edgware.RELEASE
```

Comment:
http://projects.spring.io/spring-cloud/ uses 1.5.9 for edgware

Reply:
no, 1.5.8

Reply:
I see because the `Quick Start` section it uses `.9`. The trains table is probably static and lists version from the time they were released.

Reply:
updated

Reply:
thanks, `.9` will be probably also outdated sometime soon :/

Reply:
yeah, but the most interesting when spring boot 2.0 will be released based on spring 5